src/routes/landing/components/CitySelector.js

```diff
@@ -25,11 +26,18 @@ type PropsType = {|
   cities: Array<CityModel>,
   filterText: string,
   navigateToDashboard: (city: CityModel) => void,
-  theme: ThemeType
+  theme: ThemeType,
+  currentLongitude: number | null,
+  currentLatitude: number | null,
+  renderLocationList: boolean
 |}
 
-const byName = (name: string) => {
-  return (city: CityModel) => city.name.toLowerCase().includes(name)
+const checkAliases = (a: CityModel, filterText: string): boolean => {
+  return Object.keys(a.aliases || {}).map(key => key.toLowerCase().includes(filterText.toLowerCase())).includes(true)
```

Comment:
You can use `some` instead of `map(...).includes(true)`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some